webdriver-spec.html

```diff
@@ -742,6 +742,9 @@
 
   <p>Otherwise, let <var>parameters</var> be null.</p>
 
+ <li><p>If a <a>navigation</a> is currently underway <a>wait for
+  navigation to complete</a>.
```

Comment:
@jgraham If you have further comments on this, I would suggest reading the text on master and raising a new issue before it’s too late.